packages/metrics/src/config/routes.ts

```diff
@@ -586,6 +587,16 @@ export const getRoutes = () => {
         tags: ['api']
       }
     },
+    // TODO: refactor
+    {
```

Comment:
Let's make the endpoint above this also accept requests without an auth token. The logic can be the following:

1. If auth token is present, get user id from the token & fetch practitioner id of the user
2. If auth token is not provided, then expect the caller to send an `userId` as part of the payload. Take that userId, fetch their practitionerId

Reply:
alright, will work on this